src/pages/Livestream/LiveRoomPage.css

```
@@ -1341,4 +1341,5 @@
 @keyframes lr-dot-blink {
     0%, 100% { opacity: 1; }
     50% { opacity: 0.3; }
-}
\ No newline at end of file
+}
```

Stylelint (17.7.0)

[error] Unknown word  b a c k g r o u n d - c o l o r  (CssSyntaxError)

(CssSyntaxError)